lib/mix/tasks/phoenix.gen.html.ex

```diff
@@ -66,10 +66,13 @@ defmodule Mix.Tasks.Phoenix.Gen.Html do
   end
 
   defp validate_args!([_, plural | _] = args) do
-    if String.contains?(plural, ":") do
-      raise_with_help
-    else
-      args
+    cond do
+      String.contains?(plural, ":") ->
+        raise_with_help
+      plural != Phoenix.Naming.underscore(plural) ->
+        Mix.raise "expected the second argument, `#{plural}`, to be all lowercase using snake_case convention"
```

Comment:
I don't think we use backticks in error messages in phoenix. Can you please use `inspect plural` instead?